core/prototypes/basic.py

```diff
@@ -1,29 +1,119 @@
 # -*- coding: utf-8 -*-
 from viur.core import conf
-from viur.core.skeleton import skeletonByKind
+from viur.core.skeleton import skeletonByKind, Skeleton
+from typing import Dict, List, Any, Union, Callable
 
 
 class BasicApplication(object):
 	"""
-	BasicApplication is a generic class serving as the base for the four BasicApplications.
+		BasicApplication is a generic class serving as the base for the four BasicApplications.
```

Comment:
The indent was already fine. Please use the recommended docstring format from [PEP-257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)
> The entire docstring is indented the same as the quotes at its first line

[Here](https://github.com/viur-framework/viur-core/blob/9f8fb2179a61214db04c8564bc5f676ab5c81f52/core/prototypes/list.py#L33-L48) is the indent okay (which means that it's currently totally inconsistent), but the summary is too long (should be one line). Please try to format the docstring you changed properly.

Reply:
That's how it should be indented in core (i don't mind PEP-257) and how the majority of docstrings are indented (just see above in that file)

Reply:
It seems you are right that the majority of docstrings are indented. However, the amount that is not indented is not insignificant. Almost all docstrings in the prototypes (current master) are not indented. And in many modules/files it's alternating, for instance: email, ratelimit, utils, requests.

So as we have to reindent many docstrings anyway to have a consistent layout (no matter if with or without indent), I would prefer to remove the indent and follow the PEP.